lib/src/main/java/com/knuddels/jtokkit/api/Encoding.java

```diff
@@ -47,6 +72,26 @@ public interface Encoding {
 	 */
 	List<Integer> encodeOrdinary(String text);
 
+	/**
+	 * Encodes the given text into a list of token ids, ignoring special tokens.
+	 * <p>
+	 * This method does not throw an exception if the text contains special tokens, but instead
+	 * encodes them as if they were ordinary text.
```

Comment:
As above, an additional caveat about the encoding-decoding roundtrip would be great

Reply:
I haven't mentioned the caveat we discussed, as our current solution solves it. 

Should I mention that in some cases, returned token count is smaller than expected because of multibyte characters?

Reply:
For the sake of minimizing additional roundtrips in this PR, let's leave it like this. I'll add a short sentence to the JavaDoc before I release the next version